config/options.go

```diff
@@ -404,6 +407,12 @@ func (o *Options) parsePolicy() error {
 			return err
 		}
 	}
+	for i := range o.AdditionalPolicies {
+		p := &o.AdditionalPolicies[i]
+		if err := p.Validate(); err != nil {
+			return err
+		}
+	}
```

Comment:
maybe range over o.GetAllPolicies() ?

Reply:
I tried that first, but unfortunately the `Validate` policy function actually populates needed fields in the policy and the slice is a slice of structs, not pointers, so we end up modifying a local copy of the data instead of the one in the actual slice